src/AppKit/Enums.cs

```diff
@@ -2154,10 +2154,191 @@ public enum NSPopUpArrowPosition : ulong {
 		Bottom
 	}
 
-	public static class NSFileTypeForHFSTypeCode {
```

Comment:
This type is public, which means that removing it is a breaking change.

I would just re-add it, and and add a `#if XAMCORE_4_0 ... #endif` around it (that's a define we use to mark code that we want to change if we're one day able to make breaking changes).